src/panels/NightSkyPanel/tabs/NightSkyMarkingsTab.tsx

```diff
@@ -0,0 +1,147 @@
+            identifier={'MeridianPlane'}
+          />
+        </Grid.Col>
+        <Grid.Col span={4}>
+          <IconLabelButton
+            title={'Labels'}
+            icon={'text'}
+            identifier={'AltAzGridLabels'}
+          />
+        </Grid.Col>
+        <Grid.Col span={4}>
+          <IconLabelButton title={'Zenith'} icon={'dot'} identifier={'ZenithDot'} />
+        </Grid.Col>
+      </Grid>
+      <Text my={'md'}>Ecliptic</Text>
+      <Grid columns={16} gutter={'sm'}>
+        <Grid.Col span={4}>
+          <IconLabelButton title={'Grid'} icon={'grid'} identifier={'EclipticSphere'} />
+        </Grid.Col>
+        <Grid.Col span={4}>
+          <IconLabelButton title={'Line'} icon={'line'} identifier={'EclipticLine'} />
+        </Grid.Col>
+        <Grid.Col span={4}>
+          <IconLabelButton
+            title={'Labels'}
+            icon={'text'}
+            identifier={'EclipticSphereLabels'}
+          />
+        </Grid.Col>
+        <Grid.Col span={4}>
+          <IconLabelButton title={'Band'} icon={'band'} identifier={'EclipticBand'} />
+        </Grid.Col>
+      </Grid>
+      <Text my={'md'}>Equatorial</Text>
+      <Grid columns={16} gutter={'sm'}>
+        <Grid.Col span={4}>
+          <IconLabelButton title={'Grid'} icon={'grid'} identifier={'EquatorialSphere'} />
+        </Grid.Col>
+        <Grid.Col span={4}>
+          <IconLabelButton title={'Line'} icon={'line'} identifier={'EquatorialLine'} />
+        </Grid.Col>
+        <Grid.Col span={4}>
+          <IconLabelButton
+            title={'Labels'}
+            icon={'text'}
+            identifier={'EquatorialSphereLabels'}
+          />
+        </Grid.Col>
+      </Grid>
+      <Text my={'md'}>Constellations</Text>
+      <Grid columns={16} gutter={'sm'}>
+        <Grid.Col span={4}>
+          <IconLabelButton
+            title={'Grid'}
+            icon={'grid'}
+            identifier={'ConstellationBounds'}
+          />
+        </Grid.Col>
+        <Grid.Col span={4}>
+          <IconLabelButton
+            title={'Lines'}
+            icon={'pencil'}
+            identifier={'Constellations'}
+            onAction={'os.nightsky.ShowConstellationElements'}
+            offAction={'os.nightsky.HideConstellationElements'}
+            boolProp={'Scene.Constellations.Renderable.DrawElements'}
+          />
+        </Grid.Col>
+        <Grid.Col span={4}>
+          <IconLabelButton
+            title={'Art'}
+            icon={'paint'}
+            identifier={'ImageConstellation-Ori'}
+            onAction={'os.constellation_art.ShowArt'}
+            offAction={'os.constellation_art.HideArt'}
+          />
+        </Grid.Col>
+        <Grid.Col span={4}>
+          <IconLabelButton
+            title={'Labels'}
+            icon={'text'}
+            identifier={'Scene.Constellations.Renderable.Labels'}
+            onAction={'os.nightsky.FadeInConstalltionLabels'}
+            offAction={'os.nightsky.FadeOutConstalltionLabels'}
```

Comment:
Note that there is a typo in this action identifer

Reply:
I suggested a fix in the main repo for this;  it's a typo (that should be fixed) now, but was working at the time of PR creation